community/modules/scripts/ramble-setup/main.tf

```diff
@@ -18,12 +18,15 @@ locals {
   setup_file = templatefile(
     "${path.module}/templates/ramble_setup.yml.tpl",
     {
-      install_dir = var.install_dir
-      ramble_url  = var.ramble_url
-      ramble_ref  = var.ramble_ref
-      chown_owner = var.chown_owner == null ? "" : var.chown_owner
-      chgrp_group = var.chgrp_group == null ? "" : var.chgrp_group
-      chmod_mode  = var.chmod_mode == null ? "" : var.chmod_mode
+      app_name              = "ramble"
+      profile_script        = ". {{ install_dir }}/share/ramble/setup-env.sh"
```

Comment:
You should mimic the `if [ -f ` test in the Spack solution here. Is there a reason not to?